scene/main/timer.cpp

```diff
@@ -116,16 +116,20 @@ void Timer::stop() {
 	autostart = false;
 }
 
-void Timer::set_active(bool p_active) {
-	if (active == p_active)
+void Timer::set_paused(bool p_paused) {
+	if (paused == p_paused)
 		return;
 
-	active = p_active;
+	paused = p_paused;
 	_set_process(processing);
 }
 
-bool Timer::is_active() const {
-	return active;
+bool Timer::is_paused() const {
+	return paused;
+}
+
+bool Timer::is_running() const {
+	return get_time_left() > 0;
```

Comment:
@neikeq I disagree about `!= -1`. One day a terrible platform might come out, in which `-1.0f != -1.0f`, and then we are all going to have problems...